portfolio/src/main/webapp/style.css

```diff
@@ -202,7 +203,7 @@
   #8QueenCode {
     position: relative;
   }
-  #secondBox {
+  #workBox {
```

Comment:
Some of these IDs still need to be changed from `lowerCamelCase` to `lower-hyphen-case`